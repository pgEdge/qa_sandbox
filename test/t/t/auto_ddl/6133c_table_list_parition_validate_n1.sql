SELECT pg_sleep(1);--to ensure all objects are replicated

-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6133b
SET ROLE appuser;

SET search_path TO s613, public;
--none of these should exist

\d sales_list_east
\d sales_list_west
\d sales_list_north
\d sales_list
SELECT * FROM get_table_repset_info('sales');

\d+ products_list
\d+ products_list_clothing
\d+ products_list_electronics
SELECT * FROM get_table_repset_info('products');

RESET ROLE;
--dropping the schema
DROP SCHEMA s613 CASCADE;