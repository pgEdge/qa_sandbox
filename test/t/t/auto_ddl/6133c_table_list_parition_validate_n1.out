SELECT pg_sleep(1);--to ensure all objects are replicated
 pg_sleep 
----------
 
(1 row)

-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6133b
SET ROLE appuser;
SET
SET search_path TO s613, public;
SET
--none of these should exist
\d sales_list_east
Did not find any relation named "sales_list_east".
\d sales_list_west
Did not find any relation named "sales_list_west".
\d sales_list_north
Did not find any relation named "sales_list_north".
\d sales_list
Did not find any relation named "sales_list".
SELECT * FROM get_table_repset_info('sales');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ products_list
Did not find any relation named "products_list".
\d+ products_list_clothing
Did not find any relation named "products_list_clothing".
\d+ products_list_electronics
Did not find any relation named "products_list_electronics".
SELECT * FROM get_table_repset_info('products');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

RESET ROLE;
RESET
--dropping the schema
DROP SCHEMA s613 CASCADE;
INFO:  DDL statement replicated.
DROP SCHEMA
